src/heretic/model.py

```diff
@@ -148,20 +226,51 @@ def try_add(component: str, matrix: Any):
 
         # Granite MoE Hybrid - attention layers with shared_mlp.
         with suppress(Exception):
-            try_add("mlp.down_proj", layer.shared_mlp.output_linear.weight)
+            try_add("mlp.shared_down_proj", layer.shared_mlp.output_linear.weight)
```

Comment:
Why the name distinction? I aliased multiple naming conventions together to simplify configuration and optimization. The assumption was that semantically related components would react similarly to ablation. Did you observe something different?